### PR TITLE
Remove old tokens commands

### PR DESCRIPTION
**Pull request by mrmr1993**
This PR removes the now-unused token command types in `Signed_command`.

Checklist:

- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

### CODE REVIEW

File: src/lib/transaction_snark/transaction_snark.ml [hunk 1]
```diff
@@ -2547,14 +2354,12 @@ module Base = struct
              [%with_label "Fees in tokens disabled"]
                (Boolean.Assert.is_true fee_token_default)
            ; [%with_label "Token is valid or command allows invalid token"]
-               Boolean.(Assert.any [ not token_invalid; is_create_account ])
+               Boolean.(Assert.is_true (not token_invalid))
            ; [%with_label
                "Token is default or command allows non-default token"]
```
Comment (deepthiskumar) [inline src/lib/transaction_snark/transaction_snark.ml:2359]: Safe to have `Assert.is_true token_default` here ?

File: src/lib/transaction_snark/transaction_snark.ml [hunk 2]
```diff
@@ -2565,7 +2370,6 @@ module Base = struct
                    [ not token_default
```
Comment (deepthiskumar) [inline src/lib/transaction_snark/transaction_snark.ml:2370]: safe to remove this assert?

**Review by deepthiskumar — APPROVED** [review]
Some minor comments but otherwise lgtm. Nice to see the # lines removed :))

**Comment by deepthiskumar** [comment]
I think you need to remove some stuff from src/app/cli/src/init/client.ml as well (based on dev unit tests failing)

**Comment by mrmr1993** [comment]
Stale, and these changes have made it into develop via other routes. Closing.

**Pull request closed without merging**